internal/model/metricset.go

```diff
@@ -165,9 +165,9 @@ func (me *Metricset) setFields(fields *mapStr) {
 		sample.set(name, fields)
 
 		var md mapStr
-		md.maybeSetString("type", string(sample.Type))
-		md.maybeSetString("unit", sample.Unit)
-		metricDescriptions.maybeSetMapStr(name, mapstr.M(md))
+		md.set("type", string(sample.Type))
+		md.set("unit", sample.Unit)
```

Comment:
```suggestion
		md.maybeSetString("type", string(sample.Type))
		md.maybeSetString("unit", sample.Unit)
```

Can we always send the map, but only send type or unit if non-empty? This would cut down the payload sizes a little.

Reply:
Good point! Changed!